spec/forms/legal_aid_applications/used_delegated_functions_form_spec.rb

```diff
@@ -109,6 +109,20 @@
       end
     end
 
+    context 'date is older than x months ago' do
```

Comment:
I would set this as 12 rather than x, we can always change if the logic  changes in the future